org.eclipse.buildship.ui/src/main/java/org/eclipse/buildship/ui/view/task/TaskViewContentProvider.java

```diff
@@ -71,9 +71,20 @@ public void inputChanged(Viewer viewer, Object oldInput, Object newInput) {
         // flatten the tree of Gradle projects to a list, similar
```

Comment:
This comment is no longer valid, please adjust it reflecting to the `if` statement below.